### PR TITLE
Pies & coin flip reverted + rgb colors instead of hex

### LINKED ISSUE #113: Take pie charts out of interactive map
They're cool but slowing us down. It's time to pull the plug.

### PR DESCRIPTION
**Pull request by lindsaycarr**
Fixes #113 and #131.

circlesback

Will probably have merge conflicts with #134, so will need to watch out for when that's merged.

### CODE REVIEW

**Comment by aappling-usgs** [comment]
hahahahaha `burnpies`

**Comment by lindsaycarr** [comment]
Had to keep the pie theme going until the very end

File: js/circle_utils.js [hunk 1]
```diff
@@ -1,42 +1,37 @@
...
-      .transition().delay(delay).duration(0)
-      .style("fill", categoryToColor(category) + "CC")
-      .style("stroke", categoryToColor(category)) // we should repoint this to use a lookup in styles.js
-      .style("stroke-width", "0.1");
+  // grow circles to appropriate size
+  d3.selectAll(".circle")
+    .transition().duration(1000)
+    .attr("r", function(d) { return scaleCircles(d.properties[[category]]); })
+    .style("fill", categoryToColor(category))
+    .style("stroke", categoryToColor(category)) // we should repoint this to use a lookup in styles.js
```
Comment (aappling-usgs) [inline js/circle_utils.js]: this comment is obsolete now, right?

Comment (lindsaycarr) [reply js/circle_utils.js]: oh yep! I'll get rid of that

**Pull request merged**